examples/sfinc_update_model.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Update existing Sfincs model with boundary conditions and forcing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates how to update an existing SFINCS model with gauges, boundary conditions and forcing such as precipitation. The notebook does not contain the SFINCS code or executables to run the model with. \n",
+    "\n",
+    "We will perform the following:\n",
+    "* append an existing schematization that only contains static schematization layers\n",
+    "* introduce gridded precipitation only \n",
+    "* introduce gridded precipitation, water level boundaries and observation gauge point locations\n",
+    "* write the new models to new model folders\n",
+    "\n",
+    "This notebook demonstrates the update functionalities on a test model delivered with the hydromt_sfincs plugin."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### HydroMT plugin availability\n",
+    "First let us check if HydroMT has the Sfincs plugin available. The result of the code block below should be\n",
+    "`hydroMT model plugins: sfincs (vx.x.x)` where `x.x.x` denotes the current version (e.g. `0.1.0`). If Sfincs seems missing, ensure it is installed by typing `pip install hydromt_sfincs` on a command line."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!hydromt --models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### contents of HydroMT Sfincs configuration .ini file\n",
+    "We can provide many more settings in an ini file instead of command line arguments. These settings point to the data catalog in many cases. \n",
+    "\n",
+    "###################\n",
+    "DIRK, can I refer to a notebook explaining this in more detail?\n",
+    "###################\n",
+    "\n",
+    "If you don't provide an explicit ini file, a default .ini file is used for the model building process. Let's have a look at this. "
```

Comment:
>If you don't provide an explicit ini file, a default .ini file is used for the model building process. Let's have a look at this. 

This is no longer the case. By default only the basemaps with default settings are created. An ini-file is required to build other components.